components/manager/scripts/validate.py

```diff
@@ -3,9 +3,16 @@
 import urllib2
 import platform
 import subprocess
+from os.path import join, dirname
 
 from cloudify import ctx
 
+ctx.download_resource(
+    join('components', 'utils.py'),
+    join(dirname(__file__), 'utils.py'))
+
+import utils  # NOQA
```

Comment:
Yup. It runs remotely.
On May 13, 2016 17:49, "Lukasz Maksymczuk" <notifications@github.com> wrote:

> In components/manager/scripts/validate.py
> <https://github.com/cloudify-cosmo/cloudify-manager-blueprints/pull/297#discussion_r63195934>
> :
>
> >
> >  from cloudify import ctx
> >
> > +ctx.download_resource(
> > +    join('components', 'utils.py'),
> > +    join(dirname(__file__), 'utils.py'))
> > +
> > +import utils  # NOQA
>
> can we import utils into this script?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudify-cosmo/cloudify-manager-blueprints/pull/297/files/4532c15d6e3c7ff9f86342188a8f496b7a85cd96#r63195934>
>